Source/panels/charpanel.cpp

```diff
@@ -0,0 +1,315 @@
+
+	{ "", { 135, 315 }, 33, { 0, 0 }, 0, 0, 1, false, false,
+	    []() { return colorAndText { (myPlayer._pMana != myPlayer._pMaxMana ? UiFlags::ColorRed : GetMaxManaColor()), fmt::format("{:d}", myPlayer._pMana >> 6) }; } },
+
+	{ "", { 161, 14 }, 134, { 0, 0 }, 0, 0, 1, false, false,
+	    []() { return colorAndText { UiFlags::ColorSilver, _(ClassStrTbl[static_cast<std::size_t>(myPlayer._pClass)]) }; } },
+
+	{ "experience", { 208, 52 }, 87, { -3, 0 }, 0, 0, 1, false, false,
+	    []() { return colorAndText { UiFlags::ColorSilver, fmt::format("{:d}", myPlayer._pExperience) }; } },
+
+	{ "next level", { 208, 80 }, 87, { -3, 0 }, 0, 0, 1, false, false,
+	    []() {
+	        if (myPlayer._pLevel == MAXCHARLEVEL - 1) {
+		        return colorAndText { UiFlags::ColorGold, _("None") };
+	        } else {
+		        return colorAndText { UiFlags::ColorSilver, fmt::format("{:d}", myPlayer._pNextExper) };
+	        }
+	    } },
+
+	{ "gold", { 208, 129 }, 87, { 0, -20 }, 0, 0, 1, true, false,
+	    []() { return colorAndText { UiFlags::ColorSilver, fmt::format("{:d}", myPlayer._pGold) }; } },
+
+	{ "armor class", { 250, 166 }, 45, { -3, -5 }, 55, 0, 1, false, false,
+	    []() { return colorAndText { GetValueColor(myPlayer._pIBonusAC), fmt::format("{:d}", myPlayer.GetArmor()) }; } },
+
+	{ "to hit", { 250, 194 }, 45, { -3, 0 }, 0, 0, 1, false, false,
+	    []() { return colorAndText { GetValueColor(myPlayer._pIBonusToHit), fmt::format("{:d}%", (myPlayer.InvBody[INVLOC_HAND_LEFT]._itype == ITYPE_BOW ? myPlayer.GetRangedToHit() : myPlayer.GetMeleeToHit())) }; } },
+
+	{ "damage", { 250, 222 }, 45, { -3, 0 }, 0, 0, 0, false, false,
+	    []() {
+	        std::pair<int, int> dmg = GetDamage();
+	        return colorAndText { GetValueColor(myPlayer._pIBonusDam), fmt::format("{:d}-{:d}", dmg.first, dmg.second) };
+	    } },
+
+	{ "resist magic", { 250, 259 }, 45, { -3, -5 }, 46, 1, 1, false, false,
+	    []() { return GetResistInfo(myPlayer._pMagResist); } },
+
+	{ "resist fire", { 250, 287 }, 45, { -3, -5 }, 46, 1, 1, false, false,
+	    []() { return GetResistInfo(myPlayer._pFireResist); } },
+
+	{ "resist lightning", { 250, 315 }, 45, { -3, -5 }, 76, 0, 1, false, false,
+	    []() { return GetResistInfo(myPlayer._pLghtResist); } },
+
+};
+
+Art PanelParts[6];
+Art PanelFull;
+
+void DrawPanelFieldLow(const Surface &out, Point pos, int len)
+{
+	DrawArt(out, pos.x, pos.y, &PanelParts[0]);
+	pos.x += PanelParts[0].w();
+	DrawArt(out, pos.x, pos.y, &PanelParts[1], 0, len);
+	pos.x += len;
+	DrawArt(out, pos.x, pos.y, &PanelParts[2]);
+}
+
+void DrawPanelFieldHigh(const Surface &out, Point pos, int len)
+{
+	DrawArt(out, pos.x, pos.y, &PanelParts[3]);
+	pos.x += PanelParts[3].w();
+	DrawArt(out, pos.x, pos.y, &PanelParts[4], 0, len);
+	pos.x += len;
+	DrawArt(out, pos.x, pos.y, &PanelParts[5]);
+}
+
+void DrawShadowString(const Surface &out, Point pos, panelEntry &entry)
+{
+	if (entry.label == "")
+		return;
+
+	std::string text_tmp = entry.label;

```

Comment:
Get the translated version:
```suggestion
	std::string text_tmp = _(entry.label);
```